pynecone/components/datadisplay/stat.py

```diff
@@ -9,6 +10,41 @@ class Stat(ChakraComponent):
 
     tag = "Stat"
 
+    @classmethod
+    def create(cls, *children, **props) -> Component:
+        """Create a stat component.
+
+        Args:
+            children: The children of the component.
+            props: The properties of the component.
+
+        Returns:
+            The stat component.
+        """
+        if not children:
```

Comment:
Also, should we add the keyword args in the function signature maybe - so that they're documented?
```python
def create(cls, *children, label=None, number=0, help_text=None, arrow_type=None, **props):
```

Reply:
Sure, it might be better to have the kwargs in the signature.